cirq/google/api/v2/calibration.proto

```diff
@@ -0,0 +1,100 @@
+syntax = "proto3";
+
+import "cirq/google/api/v2/metrics.proto";
+import "cirq/google/api/v2/program.proto";
+
+package cirq.google.api.v2;
+
+option java_package = "com.google.cirq.google.api.v2";
+option java_outer_classname = "BatchProto";
+option java_multiple_files = true;
+
+//  This message represents a request to execute a custom calibration routine.
+//
+message FocusedCalibration {
+     // This field represents each invocation of a calibration
+     // procedure.
+     //
+     // For instance, each (unique) moment of a circuit could be
+     // calibrated using parallel_xeb.  In this case,
+     // each moment would have its own CalibrationLayer.
+     //
+     // Some calibrations, such as readout calibration,
+     // will only have one layer.
+     repeated CalibrationLayer layers = 1;
+}
+
+// Each CalibrationLayer represents one invocation
+// of a calibration procedure.
+message CalibrationLayer {
+     // The type of the calibration procedure to execute.
+     // There will be a number of calibration types that can be run,
```

Comment:
Yes, I was going to put this in a cirq python enum.  We can point to it once it exists.

Our convention (for instance with gate sets, gate ids, and arg functions) is to use strings rather than enums.  The idea is that we then do not need to modify the public proto when we add new calibration routines (or expose them publicly for that matter).  This allows for easier augmentation.   We may add and subtract calibration procedures (fairly) frequently, just as we occasionally add new gates.

If you want, I can add a TODO to point to the cirq enum once it exists?